module/applications/mixins/sheet-v2-mixin.mjs

```diff
@@ -277,13 +279,51 @@ export default function DocumentSheetV2Mixin(Base) {
         const content = await renderTemplate("systems/dnd5e/templates/items/parts/item-summary.hbs", context);
         summary.querySelectorAll(".item-summary").forEach(el => el.remove());
         summary.insertAdjacentHTML("beforeend", content);
-        await new Promise(resolve => requestAnimationFrame(resolve));
+        await new Promise(resolve => { requestAnimationFrame(resolve); });
```

Comment:
Let's just remove this linting rule. I don't think it's ever helped flag potentially buggy behaviour for me.